### PR TITLE
Move cron jobs to `django-q` scheduled tasks

### PR DESCRIPTION
**Pull request by saadmk11**
Reference https://github.com/djangopackages/djangopackages/issues/961

TODO: 

- Remove References to cron job (we can either do this on this PR or try the new changes out first)

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,5 +1,6 @@
 bs4
 cloudflare
+croniter
```
Comment (jefftriplett) [inline requirements.in:3]: Did you want to use this for something? (Not seeing it referenced in case it's used in an unobvious place.)

**Review by jefftriplett — APPROVED** [review]
Looks good to me. I left the note about `croniter` in case you were wanting to use that in `settings`.

File: requirements.in (hunk 1 above)
Comment (saadmk11) [reply requirements.in:3]: This is required by django-q2 to parse cron expressions. https://django-q2.readthedocs.io/en/master/schedules.html#django_q.Schedule.cron

**Comment by jefftriplett** [comment]
@mashrikt looks great. We can adjust cron in a new PR. I'll hold off on deploying

**Comment by saadmk11** [comment]
> We can adjust cron in a new PR.

Do you mean the cron schedule?

---

How do we setup cron jobs on the current production servers?  I don't see any references to it on the codebase other than this:
https://github.com/djangopackages/djangopackages/blob/024ece9980e47868311f840a7daeee284925bcf2/fabfile.py#L127-L131

Do we set this up manually on production server by modifying the `crontab` file?

**Pull request merged**